doc/source/administrator/security.md

```diff
@@ -244,6 +230,51 @@ traditional computing environment), so be very careful.
 
 There's ongoing work on making this easier!
 
+## Audit Cloud Metadata server access
+
+Most cloud providers have a static IP that pods can reach to get metadata about
+the cloud. This metadata can contain very sensitive info and in the wrong hands
+allow attackers to take full control of your cluster and cloud resources. Due to
+this, it is **critical** to secure the metadata service from your user pods that
+could end up running malicious code without knowing it.
+
+The slides beginning at [_Slide
+38_](https://schd.ws/hosted_files/kccncna17/d8/Hacking%20and%20Hardening%20Kubernetes%20By%20Example%20v2.pdf)
+provides more information on the dangers presented by this attack.
+
+This Helm chart blocks access to this metadata in two ways by default, but you
+only need one.
+
+### Block metadata with a NetworkPolicy enforced by a NetworkPolicy controller
+
+If you have _NetworkPolicy controller_ such as Calico in the Kubernetes cluster,
+it will enforce the NetworkPolicy resource created by this chart that blocks
+user access to the metadata server. We recommend relying on this approach if you
+you had a NetworkPolicy controller, and then you can disable the other option.
+
+```yaml
+# default configuration
+singleuser:
+  netpol:
+    enabled: true
+```
+
+### Block metadata with a privileged initContainer running `iptables`
+
+If you can't rely on the NetworkPolicy approach to block access to the metadata
+server, we suggest relying on this option. When `singleuser.cloudMetadata.block`
+is true as it is by default, an `initContainer` is added to the user pods. It
+will run with elevated privileges and use the `iptables` command line tool to
+block access to the metadata server.
+
+```yaml
+# default configuration
+singleuser:
+  cloudMetadata:
+    block: true
```

Comment:
This is potentially confusing since if it's false cloud-metadata is still blocked if network policies are active (which is fine, I don't think it needs to be configurable since you can override the whole network policy). What do you think of changing the name of the property to e.g. `blockWithIptables` or something else?

Reply:
Nice point, I fully agree, let's be more explicit!